src/components/donorNavbar.tsx

```diff
@@ -0,0 +1,107 @@
+import {
```

Comment:
A lot of the structural elements (layout elements, i.e. the `Box` and `Flex`, these structural components will be reused for other Navbars) of this component needs to go in the `Navbar` component. ATM the `Navbar` component does absolutely nothing.

I also have a few issues with how the Navbar is styled:
1. Navbar does not span the full-width of the page
![Screen Shot 2021-11-03 at 8 59 38 PM](https://user-images.githubusercontent.com/11952174/140239800-058a31c6-ce36-4446-9af4-c5a44851a69c.png)
2. The first screenshot in #17 is a wireframe. I later added another screenshot with the high-fidelity designs for another Navbar. Please modify the navbar styles to match these hi-fi designs.
3. Please add some padding in-between items in the navbar

Reply:
Also please capitalizate the filename to `DonorNavbar.tsx`. By convention, all React components should  have a capitalized first letter. (Components in the `pages` directory are not capitalized because their filenames reflect on page routes)